bindings/wasm/manifold.d.ts

```diff
@@ -97,7 +97,7 @@ declare class Mesh {
   transform(run: number): Mat4;
 }
 
-declare class Manifold {
+export class Manifold {
```

Comment:
Maybe I was too hasty here - Doesn't this mean you're exporting `Mesh` and `Manifold` before the `Module` loads? I think that's why they're part of `ManifoldStatic`?

Reply:
nevermind, I'm pretty sure I'm wrong.

Reply:
You're actually right. It doesn't actually affect the initialization (if you do `new Manifold` you will get a runtime error), but the IDE will think that doing `new Manifold` instead of `new module.Manifold` is valid, which is a mistake. I came up with a solution and will open a new PR